tests/views/test_admin_create_repo.py

```diff
@@ -54,13 +54,16 @@ def test_create_repo(self):
             'new_repo': 'idaho'
         }, secure=True)
         # Check that the user's redirected to the repo's main admin page.
-        assert isinstance(post_resp, django.http.HttpResponseRedirect)
-        assert post_resp.url == '/idaho/admin'
+        self.assertIsInstance(post_resp, django.http.HttpResponseRedirect)
+        self.assertEqual(post_resp.url, '/idaho/admin')
         # Check that the repo object is put in datastore.
         repo = model.Repo.get_by_key_name('idaho')
-        assert repo
+        self.assertIsNotNone(repo)
+        self.assertEqual(
+            repo.activation_status, model.Repo.ActivationStatus.STAGING)
+        self.assertFalse(repo.test_mode)
         # Check a couple of the config fields that are set by default.
         repo_conf = config.Configuration('idaho')
-        assert repo_conf.language_menu_options == ['en', 'fr']
-        assert not repo_conf.launched
-        assert repo_conf.time_zone_abbreviation == 'UTC'
+        self.assertEqual(repo_conf.language_menu_options, ['en', 'fr'])
+        self.assertFalse(repo_conf.launched)
```

Comment:
supernit, self.assertIs(..., False) is more specific than self.assertFalse(...):
https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertFalse
I don't know which one is more relevant to your case though

Reply:
Thanks, changed to assertIs.